Resources/public/js/views/fields/ez-time-editview.js

```diff
@@ -104,13 +172,48 @@ YUI.add('ez-time-editview', function (Y) {
          * @return {Number}
          */
         _getFieldValue: function () {
-            var valueOfInput = this._getInputNode().get('valueAsNumber');
+            var valueOfInput,
+                time,
+                hoursTimestamp,
+                minutesTimestamp,
+                secondsTimestamp = 0;
 
-            if (valueOfInput) {
-                return valueOfInput/1000;
+            if (this.get('supportsTimeInput')) {
+                valueOfInput = this._getInputNode().get('valueAsNumber');
+                if (valueOfInput) {
+                    return valueOfInput/1000;
+                }
+                return null;
+            } else {
+
+                time = this._getInputNode().get('value').split(":");
+                hoursTimestamp = Y.Number.parse(time[0])*3600;
```

Comment:
you should first check that time has at least 2 entries and instead of using Y.Number.parse, you can use `parseInt(time[0], 10)`